tests/index.test.ts

```diff
@@ -449,6 +451,80 @@ describe('laravel-vite-plugin', () => {
             config: { delay: 123 }
         })
     })
+
+    it('configures default cors.origin values', () => {
+        const test = (pattern: RegExp|string, value: string) => pattern instanceof RegExp ? pattern.test(value) : pattern === value
+        fs.writeFileSync(path.join(__dirname, '.env'), 'APP_URL=http://example.com')
+
+        const plugins = laravel({
+            input: 'resources/js/app.js',
+        })
+        const resolvedConfig = plugins[0].config({ envDir: __dirname }, {
+            mode: '',
+            command: 'serve'
+        })
+
+        // Allowed origins...
+        expect([
+            // localhost
+            'http://localhost',
+            'https://localhost',
+            'http://localhost:8080',
+            'https://localhost:8080',
+            // 127.0.0.1
+            'http://127.0.0.1',
+            'https://127.0.0.1',
+            'http://127.0.0.1:8000',
+            'https://127.0.0.1:8000',
+            // *.test
+            'http://laravel.test',
+            'https://laravel.test',
+            'http://laravel.test:8000',
+            'https://laravel.test:8000',
+            'http://my-app.test',
+            'https://my-app.test',
+            'http://my-app.test:8000',
+            'https://my-app.test:8000',
+            'https://my-app.test:8',
```

Comment:
Should the tests also have an entry using a subdomain to satisfy the wildcard regex?

If I’m correctly reading the related regex (`/^https?:\/\/.*\.test(:\d+)?$/`), then `'https://subdomain.my-app.test',` should pass. Am I correct?

```suggestion
            // *.test
            'http://laravel.test',
            'https://laravel.test',
            'http://laravel.test:8000',
            'https://laravel.test:8000',
            'http://my-app.test',
            'https://my-app.test',
            'http://my-app.test:8000',
            'https://my-app.test:8000',
            'https://my-app.test:8',
            'https://subdomain.my-app.test',
```

Only the last line is new ☝️.

Reply:
Added this to the test suite and it is passing. Thanks!